algebras/algebra/src/test/scala/endpoints4s/algebra/client/ChunkedJsonEntitiesTestSuite.scala

```diff
@@ -1,43 +1,14 @@
 package endpoints4s.algebra.client
 
-import akka.actor.ActorSystem
-import akka.http.scaladsl.Http
-import akka.http.scaladsl.model.{ContentTypes, HttpEntity}
-import akka.http.scaladsl.server.Directives._
-import akka.http.scaladsl.server.Route
-import akka.stream.scaladsl.{Sink, Source}
-import akka.util.ByteString
 import endpoints4s.algebra.ChunkedJsonEntitiesTestApi
 
 import scala.concurrent.Future
-import scala.concurrent.duration.DurationInt
-
-trait ChunkedJsonEntitiesTestSuite[T <: ChunkedJsonEntitiesTestApi] extends ClientTestBase[T] {
 
+protected trait ChunkedJsonEntitiesTestSuite[T <: ChunkedJsonEntitiesTestApi]
+    extends ClientTestBase[T] {
```

Comment:
Split this one up because of https://github.com/endpoints4s/endpoints4s/issues/958.